climate-data-ca/resources/js/analyze-functions.js

```diff
@@ -1485,6 +1494,28 @@
                             this_has_val = true
                         }
                         break
+												
+										case 'number' :
+										
+												var this_min = parseInt($(this).attr('min')),
```

Comment:
"var" define global javascript and can easily lead to unmaintable code. Please use "let" instead